### PR TITLE
Awesomebook with ES6

### PR DESCRIPTION
**Pull request by HenaryKhuraijam**
# Awesomebook with ES6
- Created a new directory.
- Created package.json file by using npm init -y.
- Created html,css,js files.
- Created a directory called modules.
- An refactor all functions to arrow functions.
- Added a library Luxon.

### CODE REVIEW

File: modules/UI.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import { awesomeBook, saveToLocalStorage } from './DBMsystem.js';
+
+/* eslint-disable */
+const removeBtnOnClick = (e) => {
+  e.preventDefault();
+  awesomeBook.removeBook(e.target.id);
+  displayBooklist(awesomeBook.books);
+  saveToLocalStorage();
+};
```
Comment (cynthiainga) [inline modules/UI.js]: - [ ] Please kindly do not disable linters, it's good to fix our linters errors it helps us to write clean and professional codebase 🙃 😉

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @HenaryKhuraijam, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights

- [ ] Good design 👍 
- [ ] Used correctly arrow functions ✔️ 
- [ ] The app work as expected, the user can add and remove a book ✔️ 

## Required Changes ♻️

- [ ] Please kindly check the comment under the review 👇 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/UI.js (hunk 1 above)
Comment (HenaryKhuraijam) [reply modules/UI.js]: Thank you for your support.

**Comment by HenaryKhuraijam** [comment]
Made changes as suggested.

**Review by OLIPLICHE — APPROVED** [review]
Hi @HenaryKhuraijam, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights
- Applied the changes requested by the previous review 👍 
- Linter is green (the last commit has a green check mark (✅)) 👏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**